Project.toml

```
@@ -1,6 +1,6 @@
 name = "Bijectors"
 uuid = "76274a88-744f-5084-9051-94815aaf08c4"
-version = "0.15"
+version = "0.16"
```

That's just a bugfix, isn't it?
```suggestion
version = "0.15.3"
```

---

Hmm, pretty sure `truncated_inv_logabsdetjac` is new here

---

Sure, but that's part of the fix for the numerical issues of the (already existing) inverse of the `TruncatedBijector`. So IMO it's a bugfix.

Even if you'd consider it a new feature (which I'd disagree with since it's not a new bijector) then according to (Julia) semver it should be released as 0.15.3 since it's a non-breaking change. 0.16 would be a breaking release.